[2023-10-20T10:00:40.023+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=nhl_stats_save/run_id=scheduled__2023-10-19T12:00:00+00:00/task_id=save_nhl_stats permission to 509
[2023-10-20T10:00:40.059+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=nhl_stats_save/run_id=scheduled__2023-10-19T12:00:00+00:00/task_id=save_nhl_stats permission to 509
[2023-10-20T10:00:40.088+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: nhl_stats_save.save_nhl_stats scheduled__2023-10-19T12:00:00+00:00 [queued]>
[2023-10-20T10:00:40.097+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: nhl_stats_save.save_nhl_stats scheduled__2023-10-19T12:00:00+00:00 [queued]>
[2023-10-20T10:00:40.098+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-20T10:00:40.111+0000] {taskinstance.py:1382} INFO - Executing <Task(NhlStatsSaveOperator): save_nhl_stats> on 2023-10-19 12:00:00+00:00
[2023-10-20T10:00:40.119+0000] {standard_task_runner.py:57} INFO - Started process 164 to run task
[2023-10-20T10:00:40.123+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'nhl_stats_save', 'save_nhl_stats', 'scheduled__2023-10-19T12:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/nhl_stats_save.py', '--cfg-path', '/tmp/tmphru6f034']
[2023-10-20T10:00:40.125+0000] {standard_task_runner.py:85} INFO - Job 3: Subtask save_nhl_stats
[2023-10-20T10:00:40.155+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:193: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2023-10-20T10:00:40.216+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=nhl_stats_save/run_id=scheduled__2023-10-19T12:00:00+00:00/task_id=save_nhl_stats permission to 509
[2023-10-20T10:00:40.219+0000] {task_command.py:416} INFO - Running <TaskInstance: nhl_stats_save.save_nhl_stats scheduled__2023-10-19T12:00:00+00:00 [running]> on host 274b9cde8a7f
[2023-10-20T10:00:40.306+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='babintseva' AIRFLOW_CTX_DAG_ID='nhl_stats_save' AIRFLOW_CTX_TASK_ID='save_nhl_stats' AIRFLOW_CTX_EXECUTION_DATE='2023-10-19T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-19T12:00:00+00:00'
[2023-10-20T10:00:40.756+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/opt/airflow/plugins/nhl_stats_save_operator.py", line 67, in execute
    self.save_data(data)
  File "/opt/airflow/plugins/nhl_stats_save_operator.py", line 38, in save_data
    (team, stats, splits) = self.parse_data(df)
  File "/opt/airflow/plugins/nhl_stats_save_operator.py", line 29, in parse_data
    stat = {'team_uid': team_uid, 'uid': stat_uid, 'load_date': load_date} | jsStat.get('type')
TypeError: unsupported operand type(s) for |: 'dict' and 'dict'
[2023-10-20T10:00:40.765+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=nhl_stats_save, task_id=save_nhl_stats, execution_date=20231019T120000, start_date=20231020T100040, end_date=20231020T100040
[2023-10-20T10:00:40.780+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 3 for task save_nhl_stats (unsupported operand type(s) for |: 'dict' and 'dict'; 164)
[2023-10-20T10:00:40.822+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-10-20T10:00:40.842+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
